sklearn/multiclass.py

```diff
@@ -319,11 +327,27 @@ def fit(self, X, y):
             Multi-class targets. An indicator matrix turns on multilabel
             classification.
 
+        **fit_params : dict
+            Parameters passed to the ``estimator.fit`` method of each
+            sub-estimator.
+
+            .. versionadded:: 1.4
+                Only available if `enable_metadata_routing=True`. See
+                :ref:`Metadata Routing User Guide <metadata_routing>` for more
+                details.
+
         Returns
         -------
         self : object
             Instance of fitted estimator.
         """
+        _raise_for_params(fit_params, self, "fit")
+
+        routed_params = process_routing(
+            self,
+            "fit",
+            **fit_params,
+        )
```

Comment:
Let's try to use the full width when we can :)

```suggestion
        routed_params = process_routing(self, "fit", **fit_params)
```

Reply:
I let you correct the other occurences.

Reply:
I have tried, but it seems that either ruff or black corrected all but one occurrences.